webapp/src/api/api_wrapper.ts

```diff
@@ -0,0 +1,22 @@
+import Utils from 'utils';
+
+export const fetchChannels = async (mmSiteUrl: string) => {
+    const authToken = Utils.getAuthToken();
+    const url = Utils.getBaseUrls(mmSiteUrl).mattermostApiBaseUrl + '/channels?exclude_default_channels=true';
+    const response = await fetch(url, {
+        method: 'GET',
+        headers: {Authentication: authToken},
```

Comment:
```suggestion
        headers: {Authentication: Utils.getAuthToken()},
```
Remove the authToken variable